scripts/load_aks_credentials.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+# Function to check if a command exists
+not_installed() {
+    ! type "$1" &> /dev/null
+}
+
+# Check for jq
+if not_installed jq; then
+    echo "jq is not installed. You can install it from: https://stedolan.github.io/jq/download/"
+    exit 1
+fi
+
+# Check for yq
+if not_installed yq; then
+    echo "yq is not installed. You can install it from: https://github.com/mikefarah/yq"
+    exit 1
+fi
+
+# Check for op
+if not_installed op; then
+    echo "op is not installed. You can install it from: https://developer.1password.com/docs/cli/get-started/"
+    exit 1
+fi
+
+# The string to match with context.user
+USERNAME="$1"
```

Comment:
MAYBE: @IonesioJunior  could we select based on contexts instead of  username.

Generally a kubeconfig, could contain, several cluster and several users.

Same Users could have access to multiple Cluster

but a context is unique, which is a combination of a `(single cluster, single user)` which is unique among the selectable variables
Thoughts?